src/main/java/com/logdyn/ui/console/commands/HelpCommand.java

```diff
@@ -32,12 +59,22 @@ public void execute(final String... args) {
                 Thread.currentThread().interrupt();
             }
         } else {
-            System.out.println(String.format("Help page can be found at %s", HELP_URL));
+            System.out.printf("Help page can be found at %s%n", HELP_URL);
         }
     }
 
     @Override
-    public String getName() {
-        return "help";
+    public Collection<String> getNames() {
+        return Arrays.asList("help", "--help", "/?");
```

Comment:
Still not keen on /? . It's the only flag in that style, and it doesn't work in all shells.

Reply:
tough